tensorflow/core/distributed_runtime/worker.cc

```diff
@@ -25,6 +25,12 @@ limitations under the License.
 
 namespace tensorflow {
 
+// Find worker session by session handle. It's will be not null.
+template<typename Request>
+WorkerSession* Worker::FindWorkerSession(const Request* request) const {
```

Comment:
I'm not convinced that the `template` method is more readable here. Perhaps this could just take a `const string&` and the various callsites could pass `request->session_handle()`?

Reply:
But it will repeat many times for 'request->session_handle()', so I rename `FindWorkerSession(req)` to `FindWorkerSessionBy(req)`, does this improve readability?

Reply:
I think repeating a (self explanatory) accessor method call four times is still more readable than a template method. A casual reader might be confused into thinking that `RegisterGraphRequest`, `DeregisterGraphRequest`, `RunGraphRequest` etc. have a common superclass, whereas this templating only works because they share a coincidentally-named member.

Please switch `FindWorkerSession()` to taking a string.

Reply:
Looks like this comment is still not addressed?
Please correct me if I am wrong.